Dockerfile

```diff
@@ -1,14 +1,19 @@
 # Build the authorino binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 as builder
+FROM registry.access.redhat.com/ubi9-minimal:9.0.0 AS builder
 USER root
+ARG TARGETARCH
+RUN microdnf install -y tar gzip
+RUN curl -O -J "https://dl.google.com/go/go1.18.7.linux-${TARGETARCH}.tar.gz"; \
```

Comment:
I'm afraid no tag has been published yet for v1.18 at `registry.access.redhat.com/ubi{8|9}/go-toolset`, so we need to install Golang manually.